app/views/contactPreference/AddEmailAddressView.scala.html

```diff
@@ -19,46 +19,74 @@
 @import forms.YesNoForm._
 @import views.html.templates.errors.ErrorSummary
 
-@this(mainTemplate: MainTemplateOriginal, radioGroup: RadioGroup, errorSummary: ErrorSummary, formWithCSRF: FormWithCSRF)
+@this(mainTemplate: MainTemplate,
```

Comment:
All of your imports can now be removed. Three are unused, and the `FormWithCSRF` import is currently for the `play-ui` version of the form, but we should use the `govuk-frontend` one instead (which you don't need to import here)